### PR TITLE
Credit external contributors

### PR DESCRIPTION
**Pull request by robredpath**

### CODE REVIEW

**Review by BibianaC — DISMISSED** [review]

File: README.rst [hunk 1]
```diff
@@ -276,3 +275,16 @@ If the file is in spreadsheet format, the output directory will contain a *unfla
...
+===========================
+
+Development work on CoVE by Open Data Services is carried out in sprints. The issues for each sprint can be found at https://github.com/OpenDataServices/cove/projects . Other work is carried out from time to time. Outstanding issues for CoVE can be found at https://github.com/OpenDataServices/cove/issues . Please report any bugs!
+
+Community contributions to CoVE are welcome - please open an issue to discuss with the community before starting work.
+
+Credits
++++++++
+
+Thanks to `@andylolz <https://github.com/andylolz>`_ from `@pwyf <https://github.com/pwyf>`_ and `@siemvassen <https://github.com/siemvassen>`_ from `@zimmerman-zimmerman <https://github.com/zimmerman-zimmerman>`_ for code and input around CoVE for IATI, especially organisation file support.
```
Comment (andylolz) [inline README.rst]: Typo here. Should be @siemvaessen and https://github.com/siemvaessen

**Comment by andylolz** [comment]
@robredpath any progress here?

Also, CoVE still uses [bdd-tester](https://github.com/OpenDataServices/cove/blob/f4635ed2a1054ca24a5f2ef89098b6d7fe7d1823/requirements_iati.txt#L2), right?

File: README.rst [hunk 2]
```diff
@@ -276,3 +275,23 @@ If the file is in spreadsheet format, the output directory will contain a *unfla
...
+
+Credits
++++++++
+
+CoVE has been supported by open data standards organisations since its creation in 2015, as well as investment from Open Data Services. Key financial contributors have been:
+`Open Contracting Partnership <https://www.open-contracting.org/>`_
+`360Giving <http://www.threesixtygiving.org/>`_
+`Initiative For Open Ag Funding / InterAction <https://www.interaction.org/project/open-ag-funding/overview>`_
+
+As well as the team at Open Data Services, code contributions have been received from `@andylolz <https://github.com/andylolz>`_. Thanks!
```
Comment (andylolz) [inline README.rst]: > have been received from

Ummm… Not just received – they’ve been merged, in fact! [14 commits here](https://github.com/OpenDataServices/cove/commits?author=andylolz), [31 commits here](https://github.com/OpenDataServices/flatten-tool/commits?author=andylolz).

Comment (robredpath) [reply README.rst]: @andylolz I'm not sure I follow - is there a way in which a contribution could be considered to be received if it's not merged? Surely in such a case a contribution would have been proposed and then rejected (or would still be pending)?

Comment (andylolz) [reply README.rst]: You don’t propose a request – you send a request, and the other party receives a request (in English, that is. I don’t think any of this language has any special meaning in git/github).

Could you just remove the sentence, as requested by email? Thanks.

**Comment by ghost** [comment]
Closing due to conflicts, age, because the context of the request may have changed and because it references an email chain I can't check up on - but #1184 seems to say it should be?

**Pull request closed without merging**